storybookjs/storybook
# Add TypeScript support for react-scripts

Issue: #4587

## What I did
Added support for TypeScript when using `react-scripts@>=2.1.0`.

## How to test
Create a new Create React App project and pass in the `--typescript` flag.
```sh
npx create-react-app my-app --typescript
```
Install storybook (this build) and run `storybook`. You should not see any errors.
